### PR TITLE
Fixed bug in post hoc RM ANOVA for >1 RM factors

### PR DESCRIPTION
**Pull request by JohnnyDoorn**

### CODE REVIEW

File: JASP-Engine/JASP/R/anovarepeatedmeasures.R [hunk 1]
```diff
@@ -684,7 +684,7 @@ AnovaRepeatedMeasures <- function(dataset=NULL, options, perform="run", callback
     		  allEstimates <- allTees <- allSE <- allPees <- numeric() 
     		  for (k in 1:numberOfLevels) {  ### Loop over all the levels within factor and do pairwise t.tests on them
     		    for (i in .seqx(k+1, numberOfLevels)) {
-    		      tResult <- t.test(unlist(postHocData[listVarNamesToLevel[[k]]]),unlist(postHocData[listVarNamesToLevel[[i]]]), paired= T, var.equal = F)
+    		      tResult <- t.test(rowMeans(postHocData[listVarNamesToLevel[[k]]]),rowMeans(postHocData[listVarNamesToLevel[[i]]]), paired= T, var.equal = F)
```
Comment (AlexanderLyNL) [inline JASP-Engine/JASP/R/anovarepeatedmeasures.R]: Don't abbreviate boolean by a letter, write them out instead:

= TRUE
= FALSE

Especially, in your case, where F some times refers to your F-statistic

**Comment by JohnnyDoorn** [comment]
The unit test for RM ANOVA does not exist yet, so it has to be done from scratch...

File: JASP-Engine/JASP/R/anovarepeatedmeasures.R [hunk 2]
```diff
@@ -918,7 +919,7 @@ AnovaRepeatedMeasures <- function(dataset=NULL, options, perform="run", callback
 
 						row[["eta"]] <- SS / SSt
 						row[["partialEta"]] <- SS / (SS + SSr)
-						omega <- (SS - (df * MSr)) / (SSt + MSr)
+						omega <- (SS - (df * MSr)) / (SSt + MSr) #p480 Field
```
Comment (AlexanderLyNL) [inline JASP-Engine/JASP/R/anovarepeatedmeasures.R]: This still needs changing, doesn't it?

Comment (JohnnyDoorn) [reply JASP-Engine/JASP/R/anovarepeatedmeasures.R]: Updated in latest commit! Maybe @Kucharssim can recheck the calculation now?

**Pull request merged**